### PR TITLE
Wake up Dev Deployment before deploying

### PR DESCRIPTION
**Pull request by feluelle**
This enables users to create preview deployments from hibernating deployments.

### CODE REVIEW

File: action.yaml [hunk 1]
```diff
@@ -354,6 +365,11 @@ runs:
         cd ${{ inputs.root-folder }}
         astro deploy ${{steps.deployment-preview.outputs.FINAL_DEPLOYMENT_ID}} ${{steps.deploy-options.outputs.OPTIONS}}
       shell: bash
+    - name: Remove override on Deployment to resume schedule
```
Comment (sunkickr) [inline action.yaml:368]: You may want to move this step to happen after "Copy Airflow Connections, Variables, and Pools". That step requires the deployment to be awake too

File: action.yaml [hunk 2]
```diff
@@ -341,6 +341,17 @@ runs:
         echo "OPTIONS=$options" >> $GITHUB_OUTPUT
       shell: bash
       id: deploy-options
+    - name: Determine if Development Mode is enabled
```
Comment (sunkickr) [inline action.yaml:344]: I think you need to add `if [[ ${{ inputs.action }} == create-deployment-preview ]]; then` to both of these steps. The logic in these steps should only run if the action is create-deployment-preview

**Review by sunkickr — CHANGES_REQUESTED** [review]
@feluelle could you remake this PR in a branch on the main repo I gave you permission to do so

File: action.yaml (hunk 2 above)
Comment (feluelle) [reply action.yaml:344]: No, we also would want to run this logic on `deploy-deployment-preview` or `deploy` action, I think.

**Comment by feluelle** [comment]
Closing in favor of https://github.com/astronomer/deploy-action/pull/68

**Pull request closed without merging**